src/Halogen/Aff/Util.purs

```diff
@@ -6,27 +6,22 @@ module Halogen.Aff.Util
   ) where
 
 import Prelude
-
 import Control.Monad.Aff (Aff, makeAff, runAff)
 import Control.Monad.Eff (Eff)
 import Control.Monad.Eff.Class (liftEff)
 import Control.Monad.Eff.Exception (throwException, error)
 import Control.Monad.Error.Class (throwError)
 import Control.Monad.Except (runExcept)
-
-import Data.Maybe (Maybe(..), maybe)
-import Data.Either (either)
-import Data.Nullable (toMaybe)
-import Data.Foreign (toForeign)
-
 import DOM (DOM)
 import DOM.Event.EventTarget (eventListener, addEventListener)
-import DOM.HTML.Event.EventTypes (load)
 import DOM.HTML (window)
+import DOM.HTML.Event.EventTypes (load)
 import DOM.HTML.Types (HTMLElement, windowToEventTarget, htmlDocumentToParentNode, readHTMLElement)
 import DOM.HTML.Window (document)
-import DOM.Node.ParentNode (querySelector)
-
+import DOM.Node.ParentNode (QuerySelector(..), querySelector)
+import Data.Either (either)
+import Data.Foreign (toForeign)
+import Data.Maybe (Maybe(..), maybe)
 import Halogen.Aff.Effects (HalogenEffects)
```

Comment:
Our style is purely "sort the non-Prelude lines alphabetically" with no semantic understanding of the imports at all - it's just easier to maintain consistency that way, as it means you don't have to be using `psc-ide` to figure out the correct ordering... I got Christoph to explain the psc-ide ordering to me this week and I've probably forgotten it already 😜, but I think it does:

1. Open imports first
2. Qualified `as` imports next
3. Explicit imports last

It's something like that anyway. I don't know how that works when you have qualified-and-explicit imports either.